.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
     "typescript.tsc.autoDetect": "off",
     "typescript.preferences.quoteStyle": "single",
     "typescript.tsdk": "node_modules\\typescript\\lib",
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "cSpell.words": ["powerquery"]
```

Comment:
We probably want to take this out (unless you feel we should all be using the `cSpell` extension... I'm not familiar with it).

Reply:
Removing it, but I think it was a PQO recommendation to have it installed. It can be somewhat annoying, but it's also saved me from some embarrassing typos.